pmd-apex/src/main/resources/category/apex/errorprone.xml

```diff
@@ -443,4 +443,77 @@ private class TestClass {
 ]]>
        </example>
     </rule>
+    
+        <rule name="InaccessibleAuraEnabledGetter"
+          language="apex"
+          since="6.36.0"
+          message="AuraEnabled getter must be public or global if is referenced in Lightning components"
+          class="net.sourceforge.pmd.lang.apex.rule.errorprone.InaccessibleAuraEnabledGetterRule"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_apex_errorprone.html#inaccessibleauraenabledgetter">
+        <description>
+			In the Summer '21 release, a mandatory security update enforces access modifiers on Apex properties in Lightning component markup.
+			The update prevents access to private or protected Apex getters from Aura and Lightning Web Components.
+        </description>
+        <priority>3</priority>
+        <example>
+<![CDATA[// Violating
+public class Foo {
+    @AuraEnabled
+    public Integer counter { private get; set; }
+
+    @AuraEnabled
+    public static Foo bar()
+    {
+        Foo foo = new Foo();
+        foo.counter = 2; 
+        return foo;
+    }
+}
+
+public class Foo {
```

Comment:
Please split these classes into separate `<example>` elements. Also add comments to describe how these examples are relevant. It looks like this one is exactly the same thing as the previous one (modulo the inner class). This is a good test case, but it's not necessary to understand the rule, so I would remove it from the examples.

Reply:
Also please add it to the test file